package.json

```diff
@@ -28,6 +28,7 @@
     "dotenv": "^6.1.0",
     "express": "^4.16.4",
     "faker": "^4.1.0",
+    "nodemon": "^1.18.9",
```

Comment:
nodemon is usually a dev dependency but maybe its fine. i would use node server/index.js for production

Reply:
done